src/Lexer.h

```diff
@@ -71,6 +71,18 @@ void AddKeyword(night::array<Token>& tokens, night::string& token, TokenType&& t
 	token = "";
 }
 
+void easter_egg(std::string& token){
+	if(token = "squid")
```

Comment:
use `night::string` instead, and you have to use double equal signs: `==` for comp